### ISSUE TITLE
Compilation error when building python inteface

### ISSUE THREAD

**Issue by achintp** [gambitproject/gambit#159]
In building from the source I'm getting this error when trying to build the python extension

```gambit/lib/libgambit.cpp: In function ‘PyObject* __pyx_pf_6gambit_3lib_9libgambit_22EnumPureStrategySolver_4solve(__pyx_obj_6gambit_3lib_9libgambit_EnumPureStrategySolver*, __pyx_obj_6gambit_3lib_9libgambit_Game*)’:
gambit/lib/libgambit.cpp:47197:37: error: ‘>>’ should be ‘> >’ within a nested template argument list
   List<MixedStrategyProfile<Rational>>  __pyx_v_solns```

I'm running cython 0.22, gcc 4.8.2, and have python2.7-dev installed. Running on Ubuntu 14.04

Any solutions? Thanks!

**Comment by achintp** [comment]
Got it to compile by adding the padding between the braces, but am unsure if this is the expected behavior? Are we supposed to specify c++11 flag during compile time?

**Comment by tturocy** [comment]
I believe this should be corrected with commit 05c2e6f092.
